setup.py

```diff
@@ -0,0 +1,15 @@
+"""The setup for Mailroom distribution."""
+
+from setuptools import setup
+
+setup(
+    name='echo_server',
+    description='Implementation of Linked List.',
```

Comment:
Looks like a copy paste job here! Make sure to pay attention and update your documentation accordingly.